Project/Application/AppFlowCoordinator.swift

```diff
@@ -24,6 +26,7 @@ final class AppFlowCoordinator: FlowCoordinator {
 extension AppFlowCoordinator: HelloWorldViewControllerDelegate {
 
     func didSelectNextButton() {
+        let recipeRequester = RecipeRequester(with: appDependencies.appClientForRecipes)
```

Comment:
Currently, we don't need this here, so it can be removed not to cause also any merge conflicts.